submissions/FabunmiTolulopeEniola/Mygoals.txt

```diff
@@ -0,0 +1,2 @@
+I am Fabunmi Tolulope Eniola. I am so excited to be a participant of Dufuna Codecamp and I hope at the end of the 6 months: I would be a full stack developer,  I would have participated in some hackathons and successfully built an amazing online portfolio.
```

Comment:
You are yet to delete this file from your submissions folder.  Delete this file from `submissions/FabunmiTolulopeEniola`, commit your changes and push.

Reply:
I will do it immediately

Reply:
> Say for instance a Mentor comment of your PR, how do I go about it? How will the update be reflected on the Pull Request I already created? Below is a step to follow:

- Still, on that same branch, update your changes.

- Add your changes (git add .).

- Commit your changes (git commit  -m  "commit message").

- Finally, Push your change (git push origin branch_name)

Reply:
i have deleted the new branch